frontend/packages/dev-console/src/components/project-access/ProjectAccessForm.tsx

```diff
@@ -0,0 +1,185 @@
+        });
+      })
+      .catch((err) => {
+        actions.setSubmitting(false);
+        actions.setStatus({ submitError: err.message });
+      });
+  };
+
+  const handleSubmit = (values, actions) => {
+    let newRoles = getNewRoles(initialValues.projectAccess, values.projectAccess);
+    let removeRoles = getRemovedRoles(initialValues.projectAccess, values.projectAccess);
+    if (_.isEmpty(newRoles) && _.isEmpty(removeRoles)) {
+      actions.setSubmitting(false);
+      actions.resetForm({
+        values: {
+          projectAccess: initialValues.projectAccess,
+        },
+        status: { success: `Successfully updated the ${formName}.` },
+      });
+      return;
+    }
+
+    const updateRoles = getRolesWithNameChange(newRoles, removeRoles);
+    if (!_.isEmpty(updateRoles)) {
+      newRoles = _.filter(
+        newRoles,
+        (o1) => !updateRoles.find((o2) => o1.roleBindingName === o2.roleBindingName),
+      );
+      removeRoles = _.filter(
+        removeRoles,
+        (o1) => !updateRoles.find((o2) => o1.roleBindingName === o2.roleBindingName),
+      );
+    }
+
+    const roleBinding = {
+      apiVersion: 'rbac.authorization.k8s.io/v1',
+      kind: 'RoleBinding',
+      metadata: {
+        name: '',
+        namespace,
+      },
+      roleRef: {
+        apiGroup: 'rbac.authorization.k8s.io',
+        kind: 'ClusterRole',
+        name: '',
+      },
+      subjects: [
+        {
+          apiGroup: 'rbac.authorization.k8s.io',
+          kind: 'User',
+          name: '',
+        },
+      ],
+    };
+
+    const finalValues = _.isEmpty(newRoles)
+      ? values.projectAccess
+      : getFinalRoles(initialValues.projectAccess, removeRoles, newRoles);
+    actions.setSubmitting(true);
+    if (!_.isEmpty(newRoles)) {
+      sendRoleBindingRequest(Verb.Create, newRoles, finalValues, roleBinding, actions);
+    }
+    if (!_.isEmpty(removeRoles)) {
+      sendRoleBindingRequest(Verb.Remove, removeRoles, finalValues, roleBinding, actions);
+    }
+    if (!_.isEmpty(updateRoles)) {
+      sendRoleBindingRequest(Verb.Patch, updateRoles, finalValues, roleBinding, actions);
+    }
+  };
+
+  const handleReset = (values, actions) => {
+    actions.resetForm({ status: { success: null } });
+  };
+  return (
+    <React.Fragment>
+      <PageHeading title="Project Access">
+        Project Access allows you to add or remove a user&apos;s access to the project.
+      </PageHeading>
+      <hr />
+      {noProjectsAvailable ? (
+        <div className="cos-status-box">
+          <div className="text-center">No Role Bindings Found</div>
+        </div>
+      ) : (
+        <div className="co-m-pane__body">
+          <Formik
+            initialValues={initialValues}
+            onSubmit={handleSubmit}
+            onReset={handleReset}
+            validationSchema={validationSchema}
+            render={(props) => <ProjectAccessFormComponent {...props} />}
```

Comment:
You can directly say 
```suggestion
            render={(props) => <ProjectAccess {...props} />}
```